cluster/kube-up.sh

```diff
@@ -84,6 +84,7 @@ elif [[ "${validate_result}" == "2" ]]; then
 fi
 
 if [[ "${ENABLE_PROXY:-}" == "true" ]]; then
+  # shellcheck disable=SC1091
```

Comment:
why do I disable it ref: https://github.com/koalaman/shellcheck/wiki/SC1091

Reply:
Thanks for the link!

I question whether this entire block is necessary
- https://cs.k8s.io/?q=ENABLE_PROXY&i=nope&files=&repos=
- https://cs.k8s.io/?q=/tmp/kube-proxy-env&i=nope&files=&repos=

As far as I can tell, it used to be used by cluster/openstack-heat/util.sh